flexibee_backend/config.py

```diff
@@ -14,3 +14,7 @@
 
 if not FLEXIBEE_EXTERNAL_KEY_PREFIX:
     raise ImproperlyConfigured('FLEXIBEE_EXTERNAL_KEY_PREFIX must be set inside settings')
+
+FLEXIBEE_PDF_REPORT_AVAILABLE_LANGUAGES = getattr(settings, 'FLEXIBEE_PDF_REPORT_AVAILABLE_LANGUAGES',
+                                                  ('cs', 'sk', 'en', 'de'))
```

Comment:
Je to jen pojistka - když by se nááááhodou na Sečtenu zapnul nějaký jazyk, který nepodporuje flexibee